From: =?utf-8?q?Bart=C5=82omiej_Piotrowski?= <bpiotrowski@gnome.org>
Date: Wed, 9 Feb 2022 10:56:02 +0100
Subject: tests/virtme-run: Expose two CPUs

We're multi threaded, so make things more like reality by exposing more
CPUs using the '-smp 2' qemu option.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2152>
Origin: upstream, 42.1, commit:7a7fb863a2b970a0abe9a6989149e6e260146306
---
 src/tests/kvm/virtme-run.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/kvm/virtme-run.sh b/src/tests/kvm/virtme-run.sh
index f7631da..0062093 100755
--- a/src/tests/kvm/virtme-run.sh
+++ b/src/tests/kvm/virtme-run.sh
@@ -32,7 +32,7 @@ virtme-run \
   --pwd \
   --kimg "$IMAGE" \
   --script-sh "sh -c \"env $VIRTME_ENV $DIRNAME/run-kvm-test.sh \\\"$WRAPPER\\\" \\\"$WRAPPER_ARGS\\\" \\\"$TEST_EXECUTABLE\\\" \\\"$TEST_RESULT_FILE\\\"\"" \
-  --qemu-opts -cpu host,pdcm=off
+  --qemu-opts -cpu host,pdcm=off -smp 2
 
 TEST_RESULT="$(cat "$TEST_RESULT_FILE")"
 rm "$TEST_RESULT_FILE"
